### ISSUE TITLE
The points features are not displayed on the vector tiles layers

### ISSUE THREAD

**Issue by elias75015** [iTowns/itowns#1310]
iTowns do not handle the display of points feature of the vector tile Layer.

Ex. The sub stations are not displayed with iTowns, but they are with openLayers (see attached screenshots ol_vt_points.png / it_vt_points.png)
![it_vt_points](https://user-images.githubusercontent.com/6764800/73533289-c00ce680-441e-11ea-90ea-123aaaeb9f17.png)
![ol_vt_points](https://user-images.githubusercontent.com/6764800/73533290-c0a57d00-441e-11ea-8604-42da34d45d93.png)

**Comment by zarov** [comment]
There is some confusion here: there is no such thing as a `point` in vector tiles (more precisely in the style specification). You either display a [`circle`](https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#circle), or a [`symbol`](https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#symbol). iTowns currently supports `circle`, but not `symbol`, which you are currently showing here. `symbol` are being integrated in #1303.

In the past, we used to display a point for a `symbol`, but it has been removed (accidentally) in #1291. One fix that can be done is converting all your `symbol` to `circle`.
